libraries/azure_graph_users.rb

```diff
@@ -53,6 +53,8 @@ def initialize(opts = {})
 
   def guest_accounts
     @guest_accounts ||= where(userType: 'Guest').mails
+  rescue NoMethodError
```

Comment:
Would `@guest_accounts ||= where(userType: 'Guest')&.mails` be sufficient to prevent the `NoMethodError` ?

Reply:
I'll test that.

Reply:
Fount out that the safe navigator is used when the object raises 
```
NoMethodError: undefined method `[]' for nil:NilClass
```
Unfortunately, it doesn't work in that case since the raised error is 
```
NoMethodError: undefined method `mails' for #<#<Class:0x00007fc666eba490>:0x00007fc666e8b500>
```
It might be related to what `FilterTable` returns in `method_missing`.